getinsomnia/insomnia
# chore: track conflict resolution [INS-3258]

closes INS-3258

Tested locally - all 3 of the new tracked events seem to work ok.

### How to test
- Checkout this branch
- Spin up 2 instances of Insomnia (e.g. `INSOMNIA_DATA_PATH=~/Desktop/insomnia-$(date +%s) npm run dev`)
- Login with a different account on each
- Work a a shared sync collection between both accounts, try to force a conflict situation
- Check that the events are showing up on Segment Debugger for Insomnia App Dev
